x-pack/plugins/security_solution/public/detections/components/modals/ml_job_upgrade_modal/index.tsx

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiConfirmModal } from '@elastic/eui';
+import { MlSummaryJob } from '@kbn/ml-plugin/common';
+import React, { memo } from 'react';
+import styled from 'styled-components';
+import { rgba } from 'polished';
+import * as i18n from './translations';
+
+const JobsUL = styled.ul`
+  max-height: 200px;
+  overflow-y: auto;
+
+  &::-webkit-scrollbar {
+    height: ${({ theme }) => theme.eui.euiScrollBar};
+    width: ${({ theme }) => theme.eui.euiScrollBar};
+  }
+
+  &::-webkit-scrollbar-thumb {
+    background-clip: content-box;
+    background-color: ${({ theme }) => rgba(theme.eui.euiColorDarkShade, 0.5)};
+    border: ${({ theme }) => theme.eui.euiScrollBarCorner} solid transparent;
+  }
+
+  &::-webkit-scrollbar-corner,
+  &::-webkit-scrollbar-track {
+    background-color: transparent;
+  }
+`;
```

Comment:
Can't we use some EUI panel or something where this issue with scroll bars might be already addressed?

Reply:
I'm not sure on this one -- I saw this fix go into the recent value list flyout, so I imagine no, but I can check with the EUI folks and see.